tools/intersmash-tools-provisioners/src/main/java/org/jboss/intersmash/tools/provision/openshift/DBImageOpenShiftProvisioner.java

```diff
@@ -130,4 +135,20 @@ public URL getURL() {
 	public String getUrl(String routeName, boolean secure) {
 		throw new UnsupportedOperationException("Route is not created for DB applications.");
 	}
+
+	/**
+	 * When using {@link ApplicationBuilder} to build the application, then the service name defaults to the application nane
```

Comment:
What happens when `ApplicationBuilder` isn't used?

Reply:
I don't know: I just know that this is what is used with `DBImageOpenShiftProvisioner` ... perhaps the comment isn't really useful; I changed it a little

Reply:
Thanks, resolving.